types/selenium-webdriver/lib/virtual_authenticator.d.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Protocol for virtual authenticators
+ * @enum {string}
+ */
+export interface IProtocol {
+    CTAP2: string;
+    U2F: string;
+}
+
+/**
+ * AuthenticatorTransport values
+ * @enum {string}
+ */
+export interface ITransport {
+    BLE: string;
+    USB: string;
+    NFC: string;
+    INTERNAL: string;
+}
+
+export const Protocol: IProtocol;
+export const Transport: ITransport;
+
+/**
+ * Options for the creation of virtual authenticators.
+ * @see http://w3c.github.io/webauthn/#sctn-automation
+ */
+export class VirtualAuthenticatorOptions {
+    constructor();
+
+    getProtocol(): string;
+
+    setProtocol(protocol: string): void;
+
+    getTransport(): string;
+
+    setTransport(transport: string): void;
+
+    getHasResidentKey(): boolean;
+
+    setHasResidentKey(value: boolean): void;
+
+    getHasUserVerification(): boolean;
+
+    setHasUserVerification(value: boolean): void;
+
+    getIsUserConsenting(): boolean;
+
+    setIsUserConsenting(value: boolean): void;
+
+    getIsUserVerified(): boolean;
+
+    setIsUserVerified(value: boolean): void;
+
+    toDict(): Object;
+}
+
+/**
+ * A credential stored in a virtual authenticator.
+ * @see https://w3c.github.io/webauthn/#credential-parameters
+ */
+export class Credential {
+    constructor(
+        credentialId: Uint8Array,
+        isResidentCredential: boolean,
+        rpId: string,
+        userHandle: Uint8Array,
```

Comment:
```suggestion
        userHandle: Uint8Array | null,
```
The docs say that `userHandle` [may not be defined](https://w3c.github.io/webauthn/#credential-parameters), and this implementation [does apparently use null sometimes](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/virtual_authenticator.js#L150-L155).

Reply:
Change incorporated.